ambari-server/src/main/resources/Ambari-DDL-Derby-CREATE.sql

```diff
@@ -96,25 +96,6 @@ CREATE TABLE configuration_base (
   CONSTRAINT PK_configuration_base PRIMARY KEY (id)
 );
 
-CREATE TABLE clusterconfig (
```

Comment:
Should the "clusterconfig" table really be removed with this change?  

I think this might not be a good idea in the initial patch, since this table is used throughout Ambari, outside of Blueprints.